Entities/src/main/java/tw/waterball/judgegirl/entities/Question.java

```diff
@@ -31,10 +22,17 @@
     private int questionOrder;
 
     public Question(int examId, int problemId, int quota, int score, int questionOrder) {
-        this.examId = examId;
-        this.problemId = problemId;
+        this.id = new QuestionId(examId, problemId);
         this.quota = quota;
         this.score = score;
         this.questionOrder = questionOrder;
     }
+
+    @Data
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class QuestionId {

```

Comment:
rename this class to `Id`

 It's already an static Inner class under `Question`,
thus we won't lose any readability as we can refer it as `Question.Id` which is great right?